play-media-chone/04. Content Management APIs/01. Run the Mobile Application/README.md

```diff
@@ -95,3 +95,12 @@ On your physical iOS phone:
         - `npm run android`
     2. Type the "a" keyboard key to tell Expo to start the application on your Android emulator.
         - The Expo development server will install Expo Go on your Android emulator, compile the application, send it to your Android emulator, and your Android emulator will open the application.
+
+### Installing a PLAY! Media APK on a physical Android phone
```

Comment:
Please rework the whole scenario to add this section. This new way of running the application is currently inside the "Run on an Android Emulator" section which is not the right location for it. Ensure that this new section is not dependent of the repository, and Expo setup.